README.md

```diff
@@ -5,17 +5,42 @@
 > 🐋 Provide an off-chain database store for data assets metadata and registration and perform part of access control in collaboration with the keeper-contracts.
 > [oceanprotocol.com](https://oceanprotocol.com)
 
-[![Build Status](https://travis-ci.com/oceanprotocol/provider.svg?token=pA8zcB6SCxKW5MHpqs6L&branch=master)](https://travis-ci.com/oceanprotocol/provider)
+![Travis (.com)](https://img.shields.io/travis/com/oceanprotocol/provider.svg)
+![Codacy coverage](https://img.shields.io/codacy/coverage/0fa4c47049434406ad80932712f7ee6f.svg)
+![PyPI](https://img.shields.io/pypi/v/ocean-provider.svg)
+![GitHub contributors](https://img.shields.io/github/contributors/oceanprotocol/provider.svg)
 
 ---
 
 **🐲🦑 THERE BE DRAGONS AND SQUIDS. This is in alpha state and you can expect running into problems. If you run into them, please open up [a new issue](https://github.com/oceanprotocol/provider/issues). 🦑🐲**
 
 ---
 
+## Table of Contents
 
-Get Started
-------------
+  - [Features](#features)
```

Comment:
TOC needs to be updated as well. `toctoc README.md -w` if you have [toctoc](https://github.com/n1k0/toctoc) installed